plutarch-test/goldens/monadic.api.example.signatory.dev=true.uplc.golden

```diff
@@ -1,2 +1,2 @@
-do.fails (program 1.0.0 ((\i0 -> (\i0 -> (\i0 -> (\i0 -> (\i0 -> (\i0 -> \i0 -> (\i0 -> (\i0 -> (\i0 -> (\i0 -> force (i7 (equalsInteger 1 i2) (delay (force (i7 ((\i0 -> (\i0 -> (\i0 -> i2 (\i0 -> i2 i2 i1)) (\i0 -> i2 (\i0 -> i2 i2 i1))) (\i0 -> \i0 -> force (force (force chooseList) i1 (delay False) (delay (force (i10 (equalsData (i11 i1) i3) (delay True) (delay (i2 (i12 i1))))))))) (bData i6) (unListData ((\i0 -> i9 ((\i0 -> i11 (i11 (i11 (i11 (i11 (i11 (i11 i1))))))) (i11 i1))) (i8 i4)))) (delay ()) (delay error)))) (delay (force (force trace "Pattern match failure in qualified 'do' block at src/Plutarch/MonadicSpec.hs:80:5-15" (delay error)))))) (i10 i2)) (force (force fstPair) i1)) (unConstrData (i5 (i6 i1)))) (i6 i1)) #41 #d8799fd8799f9fd8799fd8799fd8799f41a0ff00ffd8799fd8799fd87a9f41a1ffd87a80ffa0d8799f41d0ffffffff80a0a141c0a149736f6d65746f6b656e018080d8799fd8799fd87980d87a80ffd8799fd87b80d87a80ffff9f45ab01fe235c4312301443abcdefff80d8799f41b0ffffd87a9fd8799fd8799f41a0ff00ffffff) (force ifThenElse)) (force headList)) (force tailList)) (\i0 -> i2 (unConstrData i1))) (force (force sndPair))))
\ No newline at end of file
+do.succeeds (program 1.0.0 ((\i0 -> (\i0 -> (\i0 -> (\i0 -> (\i0 -> (\i0 -> \i0 -> (\i0 -> (\i0 -> (\i0 -> (\i0 -> force (i7 (equalsInteger 1 i2) (delay (force (i7 ((\i0 -> (\i0 -> (\i0 -> i2 (\i0 -> i2 i2 i1)) (\i0 -> i2 (\i0 -> i2 i2 i1))) (\i0 -> \i0 -> force (force (force chooseList) i1 (delay False) (delay (force (i10 (equalsData (i11 i1) i3) (delay True) (delay (i2 (i12 i1))))))))) (bData i6) (unListData ((\i0 -> i9 ((\i0 -> i11 (i11 (i11 (i11 (i11 (i11 (i11 i1))))))) (i11 i1))) (i8 i4)))) (delay ()) (delay error)))) (delay (force (force trace "Pattern match failure in qualified 'do' block at plutarch-base/Plutarch/MonadicSpec.hs:80:5-15" (delay error)))))) (i10 i2)) (force (force fstPair) i1)) (unConstrData (i5 (i6 i1)))) (i6 i1)) #ab01fe235c #d8799fd8799f9fd8799fd8799fd8799f41a0ff00ffd8799fd8799fd87a9f41a1ffd87a80ffa0d8799f41d0ffffffff80a0a141c0a149736f6d65746f6b656e018080d8799fd8799fd87980d87a80ffd8799fd87b80d87a80ffff9f45ab01fe235c4312301443abcdefff80d8799f41b0ffffd87a9fd8799fd8799f41a0ff00ffffff) (force ifThenElse)) (force headList)) (force tailList)) (\i0 -> i2 (unConstrData i1))) (force (force sndPair))))
```

Comment:
The only difference between those is that the directory changed from `src` -> `plutarch-base` in case you're wondering

Reply:
Okay. #290 should obviate this in future.